### PR TITLE
docs: better readme to add more details

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

### Ⅰ. Describe what this PR does

This PR updates several doc files:

* README.md
* CHANGELOG.md
* FAQ.md
* ADOPTERS.md

And in README.md, I add the bi-week meeting details, update community channels and so so. 

### Ⅱ. Does this pull request fix one issue?

none

### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
no need

### Ⅳ. Describe how to verify it
none

### Ⅴ. Special notes for reviews
none

### CODE REVIEW

**Comment by allencloud** [comment]
Now we can see two more CI validation there: 
* ci/circleci: code-check;
* ci/circleci: markdownlint-misspell-shellcheck

**Comment by jian-he** [comment]
lgtm, conflicts need to be fixed

File: README.md [hunk 1]
```diff
@@ -158,10 +160,17 @@ Generate manifests e.g. CRD, RBAC etc.
...
+If you have any questions or want to contribute, you are welcome to communicate most things via GitHub issues or pull requests.
+
+Other active communication channels:
+
+- Slack: [channel address](https://join.slack.com/t/kruise-workspace/shared_invite/enQtNjU5NzQ0ODcyNjYzLWMzZDI5NTM3ZjM1MGY2Mjg1NzU4ZjBjMDJmNjZmZTEwYTZkMzk4ZTAzNmY5NTczODhkZDU2NzVhM2I2MzNmODc)
+- Mailing List: todo
+- Dingtalk Group(钉钉讨论群)
 
-Mailing List: todo
+<div align="center">
```
Comment (xiang90) [inline README.md]: change this to markdown instead of using html?

Comment (allencloud) [reply README.md]: I wish to, while actually I found no way to support that in the github markdown.

**Comment by allencloud** [comment]
Thanks to your review @jian-he @xiang90 
I will merge this first.

**Pull request merged**